doc/Makefile.am

```diff
@@ -93,12 +93,13 @@ publican_docs	+= $(docbook)
 endif
 endif
 
-EXTRA_DIST	= $(ascii) $(SHARED_TXT) $(PNGS_ORIGINAL) $(SVGS)
+EXTRA_DIST	= $(ascii) $(SHARED_TXT) $(PNGS_ORIGINAL) $(DOTS) $(SVGS)
```

Comment:
What's the point of shipping dot files when they are only ever the
intermediate byproducts in the documentation context and given that
the derived files are (likely) shipped as well?

Distribution tarballs don't aim to prove the whole chain of
intermediaries till the final artefacts are reached.

Reply:
Oh, I see the context now, dots are not generated, there's a TODO
for that.